sections/04_study.md

```diff
@@ -0,0 +1,90 @@
+## How is deep learning used to study basic biological processes in a manner that may provide future insights into human disease?
+
+*The (awkward) placeholder section title is intended to help define the scope.
+We do not want this section to become a miscellaneous collection of everything
+that does not fit in Categorize and Treat.*
+
+*One proposal is that we organize this roughly by what is being predicted,
+which will generally correspond to the types of data being used.  For each
+sub-section we can quickly introduce the prediction problem and cite some
+examples of the relevance to disease.  Hypothetically, if we had an algorithm
+that produced perfect predictions on the task, what would we learn and how
+could those predictions be used?*
+
+*Existing reviews could be mentioned briefly.*
+
+*It may not fit here, but there could be a general discussion of why different
+neural network architectures are particularly well-suited for different types
+of input data.  For example, CNNs and RNNs for 1-dimensional data are used
+in several categories below.*
+
+*A few suggestions for sub-sections follow.  Some of these could be left out
+because our goal is not an exhaustive enumeration of methods.  Some
+are important areas of biology, but there may not be much deep learning-
+specific content to present.  Others may be important areas where we lack
+expertise, in which case we may acknowledge the application area but not
+dive into merits or weaknesses of individual methods.*
+
+### Gene expression
```

Comment:
If you want to assign these bits out like you did with the metagenomics section, you can assign this and splicing to me. I may see if we can snag a splicing guru from Penn (Yoseph Barash). If not, I did read a few of those papers and could write that bit.